src/entry/mod.rs

```diff
@@ -0,0 +1,15 @@
+use hopper;
+use metric::Event;
+
+pub trait Entry {
```

Comment:
Actually, we don't need the full config, only "forwarding" sections. So I should simplify "Entry" to contain only copy of it.

About simplifications in testing. 
In the `master` the approach is to go all the sinks/sources/filters one by one, creating the threads. So we can't test it.
In my approach I've packed it into two separate cycles.
Yes, it is still impossible to test, but it is only a first step.

The next step would be to have 3 cycles: 
- to initialyze data
- given graphs of "followers"  make some conclusions 
- run threads.
If the first two cycles are in function - it makes it very easy to test.
I should have done it in this PR, I agree, so I'll do it here.